packages/grafana-ui/src/components/Collapse/CollapsableSection.tsx

```diff
@@ -27,22 +27,22 @@ export const CollapsableSection: FC<Props> = ({ label, isOpen, children }) => {
   );
 };
 
-const collapsableSectionStyles = (theme: GrafanaTheme) => {
+const collapsableSectionStyles = (theme: GrafanaThemeV2) => {
   const header = css({
     display: 'flex',
     justifyContent: 'space-between',
     fontSize: theme.typography.size.lg,
-    padding: `${theme.spacing.xs} 0`,
+    padding: `${theme.spacing(4)} 0`,
```

Comment:
xs should be 0.5 ?

Reply:
yeah...you're right.
would fix up right away.